salt/modules/crmsh.py

```diff
@@ -9,3 +9,53 @@
 :depends: crmsh
 '''
 from __future__ import absolute_import
+from salt.ext import six
+import re
+
+
+def configure_show(xml=False, changed=False, show_type=None):
+    '''
+    Display CIB objects
+
+    xml
+        show CIB objects with XML format (default: False)
+    changed
+        show all modified objects (default: False)
+    show_type
+        list of show type, valid elements include:
+        object IDs,
+        object type, use the \'type:\' prefix,
+        object tag, use the \'tag:\' prefix,
+        constraints related to a primitive, use the \'related:\' prefix.
+        (default: None)
+
+        type :: node | primitive | group | clone | ms | rsc_template | bundle
+              | location | colocation | order
+              | rsc_ticket
+              | property | rsc_defaults | op_defaults
+              | fencing_topology
+              | role | user | acl_target
+              | tag
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' crmsh.configure_show 
+        salt '*' crmsh.configure_show show_type="vip, type:node"
+        salt '*' crmsh.configure_show xml=True show_type='related:vip'
+        salt '*' crmsh.configure_show changed=True
```

Comment:
How about simplifying a bit, and just calling it `show` instead of `configure_show`?

Also, `show_type` is a bit redundant when there is a `type:` filter in the command itself. How about calling it `filter`?

```
salt '*' crmsh.show filter="vip, type:node"
```

Otherwise it looks like a great start!